src/qtgui/plotter.cpp

```diff
@@ -1554,9 +1585,8 @@ int CPlotter::xFromFreq(qint64 freq)
 // Convert from screen coordinate to frequency
 qint64 CPlotter::freqFromX(int x)
 {
-    qint64 w = width();
-    qint64 StartFreq = m_CenterFreq + m_FftCenter - m_Span / 2;
-    qint64 f = StartFreq + m_Span * (qint64) x / w;
+    double ratio = (double)x / (double)width();
+    qint64 f = (m_CenterFreq + m_FftCenter - m_Span / 2) + ratio * m_Span;
```

Comment:
What is the rationale behind this change?

Reply:
I guess it's meant to match the other `ratio` calculations, which makes sense. :+1: